environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -93,6 +94,13 @@ SimpleEnvironmentResponses deleteMultipleByNames(Set<String> names,
     @ApiOperation(value = EnvironmentOpDescription.LIST, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES, nickname = "listEnvironmentV1")
     SimpleEnvironmentResponses list();
 
+    @GET
+    @Path("internal")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.INTERNAL_LIST, produces = MediaType.APPLICATION_JSON, notes = ENVIRONMENT_NOTES,
+            nickname = "internalListEnvironmentV1")
+    SimpleEnvironmentResponses listInternal(@QueryParam("accountId") @AccountId String accountId);
```

Comment:
@horadla23 do we need to annotate these query parameters @NotEmpty?

Reply:
no, but you need to add '@AccountId' here too since it is a validation annotation

Reply:
oh sorry it is already there, then this should be ok

Reply:
ok. thanks! I didn't know if @accountId implied @notempty.

Reply:
https://github.com/hortonworks/cloudbreak/blob/master/auth-internal-api/src/main/java/com/sequenceiq/cloudbreak/auth/security/internal/AccountIdValidator.java validation checks also if string is blank or not so it should be enough